src/cards/FanCard.ts

```diff
@@ -0,0 +1,43 @@
+import {AbstractCard} from "./AbstractCard";
+import {cards} from "../types/cards";
+import {generic} from "../types/generic";
+
+// noinspection JSUnusedGlobalSymbols Class is dynamically imported.
```

Comment:
Would it be worth considering not having dynamically imported classes? Do we really gain a lot performance wise?
If we removed the dynamic imports, we would get rid of these "noinspections" as well as make the code easier to understand and maintain.

Reply:
Since we don't always know which modules we require on beforehand, hence the dynamic imports.
Domains, Chips, Areas are configurable which changes the import requirement.

Also, I'm not keen of importing modules if there's no need for the module to be imported yet.
I prefer something like "Just in time" imports.